contribs/gmf/src/search/component.js

```diff
@@ -321,7 +321,7 @@ gmf.search.component.SearchController_ = class {
      * @type {number}
      * @export
      */
-    this.delay = parseInt(this.scope_['delay'], 10) || 50;
+    this.delay = 50;
```

Comment:
You should not assign to `this.delay` in the constructor. The value will be lost.

Reply:
no